src/metabase/search/spec.clj

```diff
@@ -288,7 +289,10 @@
 (defn- instance->db-values
   "Given a transformed toucan map, get back a mapping to the raw db values that we can use in a query."
   [instance]
-  (let [xforms (#'t2.transformed/in-transforms (t2/model instance))]
+  (let [xforms (try
+                 (#'t2.transformed/in-transforms (t2/model instance))
+                 (catch Exception _     ; this happens for :model/ModelIndexValue, which has no transforms
```

Comment:
I don't know how to check if there is not going to be an exception, it's bloody hard to get into `t2.transformed/transforms` internals.